storyblok-vercel-configuration.md

```diff
@@ -0,0 +1,111 @@
+# Production, Staging, Preview configuration
+
+Storyblok = place where content is stored (Headless CMS = content management
+system)
+
+GitHub (this repo) = place where code is stored (hosted SVC = software version
+control)
+
+Vercel/Next.js = service that generates and deploys website by combining content
+and code.
+
+The following document uses Quansight Consulting "LLC" as an example, but it
+also applies to Labs: wherever you come across the string `consulting` below,
+you just have to replace it with `labs`.
+
+## Production = Live Public-Facing Site
+
+Currently deploys to: quansight-consulting.vercel.app
+
+Later, will deploy to: quansight.com
+
+Triggered by:
+
+- adding new commits (e.g. merging a PR) to `main` branch of [Quansight-website
+  repo](https://github.com/Quansight/Quansight-website)
+- Promote to Production button in vercel.com admin
+
+Behind the scenes: Vercel fetches all of the Published content from Storyblok
+and generates a static site using the code on `main`.
+
+## Staging = Semi-Public Preview Site
+
+Semi-public = anybody with the url can see the preview site so be careful not to
+publish anything sensitive or prematurely to staging!
+
+Deploys to: quansight-consulting-git-develop-quansight.vercel.app
+
+Triggered by:
+
+- Publish/unpublish hook on Storyblok (this means the Publish button in
+  top-right of the UI)
+- Commit/merge to `develop` branch
+
+Behind the scenes: Vercel fetches all of the Published content from Storyblok
+and generates a static site using the code on `develop`.
+
+## Private Preview
+
+> Be careful not to share preview URLs because they contain secrets.
+
+This is for changes in Storyblok that have been **saved** but **not published**.
+
+Production preview = Published + Saved (not Published) Storyblok content + code
+on `main` branch:
+https://quansight-consulting.vercel.app/api/preview?secret=SECRET&slug=
```

Comment:
My understanding is that if you go to this URL without filling in the slug, it will take you to the homepage and set a cookie so that as you navigate through the web site, you will see preview content, is that correct?